src/git/git.go

```diff
@@ -447,6 +453,29 @@ func (g *GitVersioning) initialize() error {
 	return nil
 }
 
+func newVersion(tag string) *Version {
+	segments := strings.Split(tag, ".")
+	major, _ := strconv.Atoi(segments[0])
+	minor, _ := strconv.Atoi(segments[1])
+	patch, _ := strconv.Atoi(segments[2])
```

Comment:
do you need check segments length? @luc-zago

Reply:
@LeoCBS I think there is no need for it since there is a pattern check through regex [here](https://github.com/NeowayLabs/semantic-release/blob/main/src/git/git.go#L240) and [here](https://github.com/NeowayLabs/semantic-release/blob/main/src/git/git.go#L26). What do you think?

Reply:
sure :) @luc-zago